buildSrc/DesktopBuilder.js

```diff
@@ -125,32 +125,39 @@ async function rollupDesktop(dirname, outDir, version, platform, disableMinify)
 			}),
 			resolveLibs(),
 			nativeDepWorkaroundPlugin(),
-			keytarNativeBannerPlugin({
-				rootDir: projectRoot,
-				platform,
-			}),
 			nodeResolve({
 				preferBuiltins: true,
 				resolveOnly: [/^@tutao\/.*$/, "keytar"],
 			}),
-			// requireReturnsDefault: "preferred" is needed in order to correclty generate a wrapper for the native keytar module
+			// requireReturnsDefault: "preferred" is needed in order to correctly generate a wrapper for the native keytar module
```

Comment:
for keytar specifically it looks odd because we unpack it immediately anyway?

Reply:
i just got that keytar is not cjs anymore now and that's probably why i had problems with that import

Reply:
scratch that, `requireReturnsDefault` applies to cjs modules importing es modules, not the other way around